airbyte-scheduler/src/main/java/io/airbyte/scheduler/persistence/DefaultJobCreator.java

```diff
@@ -124,7 +125,7 @@ public long createGetSpecJob(String integrationImage) throws IOException {
   }
 
   // Strategy:
-  // 1. Set all streams to full refresh.
+  // 1. Set all streams to full refresh - overwrite.
   // 2. Create a job where the source emits no records.
   // 3. Run a sync from the empty source to the destination. This will overwrite all data for each
   // stream in the destination.
```

Comment:
Instead of an empty source, could we force a full refresh from source instead?
https://github.com/airbytehq/airbyte/issues/2484